pages/_app.tsx

```diff
@@ -122,18 +123,18 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
               right: 0,
               top: 0,
               minHeight: '768px',
+              aspectRatio: '1',
               backgroundImage: `url('${process.env.BASE_PATH}/img/page-background.png')`,
               backgroundSize: 'cover',
               backgroundPosition: 'center top',
-              '@media (min-width: 1440px)': {
-                aspectRatio: '1440/768',
-              },
             }}
           />
```

Comment:
This changes what the page looks like:

**Before**
<img width="794" alt="CleanShot 2022-08-05 at 17 55 39@2x" src="https://user-images.githubusercontent.com/1059139/183216502-fa55adb1-9214-4877-87d7-27f54b96240a.png">

**After**
<img width="786" alt="CleanShot 2022-08-05 at 17 56 39@2x" src="https://user-images.githubusercontent.com/1059139/183216554-ff3febe1-07e4-4a01-a9c1-c8383aacac21.png">

I think a better solution would be to wrap this `div` in another position-absolute `div` that has `overflow: hidden`.

Reply:
Yes, that's true. That does work well, I have updated this. Thank you. :)